ssv/spectest/tests/runner/postconsensus/invalid_and_valid_validator_indexes_quorum.go

```diff
@@ -0,0 +1,80 @@
+package postconsensus
+
+import (
+	"github.com/ssvlabs/ssv-spec/ssv/spectest/tests"
+	"github.com/ssvlabs/ssv-spec/types"
+	"github.com/ssvlabs/ssv-spec/types/testingutils"
+)
+
+// InvalidAndValidValidatorIndexesQuorum tests a quorum of post consensus messages with both an invalid and a valid validator index
+func InvalidAndValidValidatorIndexesQuorum() tests.SpecTest {
+
+	ks := testingutils.Testing4SharesSet()
+	expectedError := "could not find validators for root"
+	return &tests.MultiMsgProcessingSpecTest{
+		Name: "post consensus invalid and valid validator index quorum",
+		Tests: []*tests.MsgProcessingSpecTest{
+			{
+				Name: "attester",
+				Runner: decideCommitteeRunner(
+					testingutils.CommitteeRunner(ks),
+					testingutils.TestingAttesterDuty,
+					&testingutils.TestBeaconVote,
+				),
+				Duty: testingutils.TestingAttesterDuty,
+				Messages: []*types.SignedSSVMessage{
+					testingutils.SignPartialSigSSVMessage(ks, testingutils.SSVMsgCommittee(ks, nil, testingutils.PostConsensusInvalidThenValidValidatorIndexAttestationMsg(ks.Shares[1], 1, testingutils.TestingDutySlot))),
+					testingutils.SignPartialSigSSVMessage(ks, testingutils.SSVMsgCommittee(ks, nil, testingutils.PostConsensusInvalidThenValidValidatorIndexAttestationMsg(ks.Shares[2], 2, testingutils.TestingDutySlot))),
+					testingutils.SignPartialSigSSVMessage(ks, testingutils.SSVMsgCommittee(ks, nil, testingutils.PostConsensusInvalidThenValidValidatorIndexAttestationMsg(ks.Shares[3], 3, testingutils.TestingDutySlot))),
+				},
+				OutputMessages: []*types.PartialSignatureMessages{},
+				BeaconBroadcastedRoots: []string{
+					testingutils.GetSSZRootNoError(testingutils.TestingSignedAttestation(ks)),
+				},
+				DontStartDuty: true,
+				ExpectedError: expectedError,
```

Comment:
Returning an error usually means that the execution failed because something unexpected happened.

(for example os.Open would only return an error if the file did not open for some unexpected reason)

I'd suggest to not return an error and instead check that the submitted attestations are only for the known validators.